app/components/course-stage-screencast-preview.ts

```diff
@@ -0,0 +1,22 @@
+import Component from '@glimmer/component';
+import type CourseStageScreencastModel from 'codecrafters-frontend/models/course-stage-screencast';
+
+interface Signature {
+  Element: HTMLDivElement;
+
+  Args: {
+    screencast: CourseStageScreencastModel;
+  };
+}
+
+export default class CourseStageScreencastPlayer extends Component<Signature> {
+  get roundedDurationToNearestMinute() {
```

Comment:
Grammatically this name doesn't make sense - it's a valid "sentence" but not a valid "object". Also it removes the "InSeconds", so it's isn't clear what unit the new value is in - is it minutes? is it seconds?

Some valid alternatives: 

- `durationInSecondsRoundedToNearestMinute`
- `durationInSecondsForDisplay`
- `roundedDurationInSeconds`

My thought process: The #1 aspect to convey using this name is that it is a number and unit is seconds. #2 is how it is different from `@screencast.durationInSeconds` - in this case, it's because it's rounded / modified in some way for display purposes.